RosBE-Unix/Base-i386/RosBE-Builder.sh

```diff
@@ -171,13 +171,13 @@ echo "Using CXXFLAGS=\"$CXXFLAGS\""
 echo
 
 if $rs_process_cpucount; then
-	rs_do_command $CC -s -o "$rs_prefixdir/bin/cpucount" "$rs_scriptdir/tools/cpucount.c"
+	rs_do_command $CC $CFLAGS $LDFLAGS -s -o "$rs_prefixdir/bin/cpucount" "$rs_scriptdir/tools/cpucount.c"
```

Comment:
`CFLAGS` and `LDFLAGS` are exported, so they should be picked up by the host compiler anyway.
Apart from that, you can't just pass `LDFLAGS` to the C compiler.